### ISSUE TITLE
support for includes gone in distill_article?

### ISSUE THREAD

**Issue by wcmbishop** [rstudio/distill#317]
I previously used the `includes` input in `distill_article()` to pass in additional html content (e.g. custom header/footer) using the `rmarkdown::includes()` function. I still see `includes` listed in the `distill_article()` documentation, but the code doesn't seem to use that input anywhere. I don't see any content passed in via `includes` in the knitted html.

**Comment by cderv** [comment]
Do you have an example of it not working ? I can't reproduce on my side. 

The code use the includes provided in `distill_article` format processing (https://github.com/rstudio/distill/blob/master/R/distill_article.R)

Thanks.

**Comment by jmbuhr** [comment]
I seem to find the opposite of @wcmbishop's issue. In my example, the header is included twice. I was trying to implement a darkmode button for https://github.com/rstudio/distill/issues/272 when I noticed that the code shows up twice in the final index.html.

I am using

```
output:
  distill::distill_article:
    includes:
      in_header: "_header.html"
```
`_header.html` has a script tag and for testing a single `<p>hi</p>`.
Everything turns up twice when I inspect index.html in the browser.

**Comment by jjallaire** [comment]
That issue (duplicate includes) should be fixed in v1.2 of distill (now on
CRAN):
https://github.com/rstudio/distill/commit/c272438210377d1e7b0e05a974c60055bc5d8453

**Comment by jmbuhr** [comment]
Oh, indeed. Thank you!

**Comment by cderv** [comment]
@wcmbishop it should be resolved in 1.2. Please reopen if you still have some issues. Thank you !
